app/scripts/migrations/index.js

```diff
@@ -156,6 +156,7 @@ const migrations = [
   require('./132'),
   require('./133'),
   require('./134'),
+  require('./135'),
```

Comment:
The latest migration version in branch `Version-v12.9.2` is `133`.  It might be easier to cherry pick if we named this one `133.1` instead.

Reply:
aaah yes that's a good point